runner/cli.ts

```diff
@@ -7,18 +7,19 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
-var _      = require('lodash');
-var chalk  = require('chalk');
-var events = require('events');
+import * as _ from 'lodash';
```

Comment:
Sorted all imports. Annoyingly, I couldn't find a sort-imports lint checker that does what we want here.